src/translator/response_builder.h

```diff
@@ -23,11 +23,13 @@ class ResponseBuilder {
   /// or not in the response and any additional configurable parameters.
   /// @param [in] vocabs: marian vocab object (used in decoding)
   /// @param [in] promise: promise to set with the constructed Response.
+  /// @param [in] qualityEstimator: promise to set with the constructed Response.
   ResponseBuilder(ResponseOptions responseOptions, AnnotatedText &&source,
                   std::vector<Ptr<Vocab const>> &vocabs,
-                  std::promise<Response> &&promise)
+                  std::promise<Response> &&promise, 
+                  AlignedMemory &qualityEstimator)
       : responseOptions_(responseOptions), source_(std::move(source)),
-        vocabs_(&vocabs), promise_(std::move(promise)) {}
+        vocabs_(&vocabs), promise_(std::move(promise)), qualityEstimator_(&qualityEstimator) {}
```

Comment:
`QualityEstimator` doesn't have a constructor yet which accepts `AlignedMemory`, does it? I hope it's in the works.

Reply:
done:  064cfe3